packages/utils/parse.ts

```diff
@@ -0,0 +1,47 @@
+import { LocationNames } from "./constants";
+import { CampusDishResponse } from "../validators/src";
```

Comment:
Its good practice to access the packages/apps through @acme/<package-name> so we can take advantage of the pnpm dependency management

Reply:
When I try using the alias I get an error and it doesn't seem to recognize @acme/api or @acme/utils. Does each subdir need some kind of config or something? I'm pretty unfamiliar with this

Reply:
You should be able to add it to the package dependencies by running pmpm add @acme/api

Reply:
Make sure you your working directory is in the package you wanna add it to!